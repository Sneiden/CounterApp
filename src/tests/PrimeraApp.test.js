import React from 'react';
// import { render } from '@testing-library/react';
import PrimeraApp from '../PrimeraApp';
import { shallow } from 'enzyme'
import '@testing-library/jest-dom'

describe('Pruebas en <PrimeraApp />', () => {

    // test('debe de mostrar el mensaje "Hola Mundo"', () => {
    //     const saludo = 'Hola Mundo'
    //     const {getByText} = render( <PrimeraApp saludo={ saludo }/>)
    //     expect( getByText( saludo ) ).toBeInTheDocument();
    // })

    test('debe de mostrar <PrimeraApp />', () => { 

        const saludo = 'Hola Mundo'
        const wrapper = shallow( <PrimeraApp saludo={ saludo }/> )

        expect( wrapper ).toMatchSnapshot();

    })

    test('debe de mostrar el subtitulo enviado por props', () => { 
        
        const saludo = 'Hola Mundo'
        const subtitulo = 'soy un subtitulo'

        const wrapper = shallow( 
            <PrimeraApp 
                saludo={ saludo }
                subtitulo={ subtitulo }
            />
        );

        const textoParrafo = wrapper.find('p').text();

        expect( textoParrafo ).toBe( subtitulo );

    })

})