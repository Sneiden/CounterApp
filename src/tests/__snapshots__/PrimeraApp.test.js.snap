// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe de mostrar <PrimeraApp /> 1`] = `
<Fragment>
  <h1>
    Hola Mundo
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
